srcpkgs/NetworkManager-openconnect/template

```diff
@@ -1,17 +1,18 @@
 # Template file for 'NetworkManager-openconnect'
 pkgname=NetworkManager-openconnect
-version=1.2.6
-revision=4
+version=1.2.10
+revision=1
+archs="i686 x86_64 x86_64-musl"
 build_style=gnu-configure
-configure_args="--disable-static"
+configure_args="--disable-static --with-gtk4"
 hostmakedepends="pkg-config intltool"
-makedepends="gcr-devel gtk+3-devel liblz4-devel libnma-devel
- openssl-devel libsecret-devel libxml2-devel openconnect-devel"
+makedepends="gcr-devel gtk4-devel gtk+3-devel liblz4-devel libnma-devel
```

Comment:
gtk3 and gtk4?

Reply:
I started with the old template and forgot to check if that was actually necessary